product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/03_pgbouncer_configuring.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuring EDB PgBouncer
```

Comment:
use this as the navTitle and Configuring EDB PgBouncer for BigAnimal as the topic title, then you can add a link to the topic from the parent page.